### PR TITLE
Refactor code

### PR DESCRIPTION
**Pull request by nasirhemed**
- [Move encoder command functions to a different file.](https://github.com/googleinterns/av1-codec-comparison/commit/45afbaa8f4e931a59b566f6a40b539b8b17c7a2b)
    -   Since we will be using a lot of different encoder commands,
we will keep a separate file to just store all the encoder
command configurations and import them into the main script.
- [Use latest version of aom and add setup scripts for rav1e and svt.](https://github.com/googleinterns/av1-codec-comparison/commit/a614768fff380e51c98ce84511b406426f2a9209)
    - Given our project will be using the open source tools for av1, add setup
scripts for rav1e and svt so that we can use them for metric analysis.
The setup script for aom checked out a very old commit and was not able to
decode the videos encoded by rav1e and svt. Thus, we updated the git
checkout to one of the recent versions and modified the setup commands
to reflect the current way of setting up aom.
- [Format code using yapf](https://github.com/googleinterns/av1-codec-comparison/commit/7fcbd8f019fadec8844021a7d00547eaa2503784)
- [Refactor string binaries used and store them in binary_vars.py.](https://github.com/googleinterns/av1-codec-comparison/commit/a8837943ca74c06d28c752c1f1d182fe4c09b5b9)
   - Since the locations of the binaries that are installed in the setup
scripts such as the encoder/decoder commands may change, it will be
better to have them stored as constants so that they can be used by the
script and the encoder_command functions

### CODE REVIEW

File: rtc-video-quality/binary_vars.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+# limitations under the License.
+
+# Encoder Binaries
+AOM_ENC_BIN = 'aom/aom_build/aomenc'
+VPX_ENC_BIN = 'libvpx/vpxenc'
+H264_ENC_BIN = 'openh264/h264enc'
+YAMI_ENC_BIN = 'yami/libyami/bin/yamiencode'
+VPX_SVC_ENC_BIN = 'libvpx/examples/vpx_temporal_svc_encoder'
+
+# Decoder Binary
```
Comment (wachsler-google) [inline rtc-video-quality/binary_vars.py:22]: s/Binary/Binaries/

File: rtc-video-quality/binary_vars.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+AOM_DEC_BIN = 'aom/aom_build/aomdec'
+VPX_DEC_BIN = 'libvpx/vpxdec'
+H264_DEC_BIN = 'openh264/h264dec'
+
+# Metrics Binary
```
Comment (wachsler-google) [inline rtc-video-quality/binary_vars.py:27]: Here too.

File: rtc-video-quality/encoder_commands.py [hunk 3]
```diff
@@ -16,6 +16,7 @@
 import os
 import subprocess
 import tempfile
+from binary_vars import *
```
Comment (wachsler-google) [inline rtc-video-quality/encoder_commands.py:19]: Use import statements for packages and modules only, not for individual classes or functions.
http://google.github.io/styleguide/pyguide.html#22-imports

File: rtc-video-quality/encoder_commands.py [hunk 4]
```diff
@@ -36,7 +36,7 @@ def aom_command(job, temp_dir):
     clip = job['clip']
     fps = int(clip['fps'] + 0.5)
     command = [
-      AOM_ENC_BIN,
+      binary_vars.binary_vars.AOM_ENC_BIN,
```
Comment (wachsler-google) [inline rtc-video-quality/encoder_commands.py:39]: You shouldn't need "binary_vars.binary_vars." here. Just one should suffice.

Comment (nasirhemed) [reply rtc-video-quality/encoder_commands.py:39]: Oh sorry about this mistake. I'll update it

**Comment by nasirhemed** [comment]
> In commit message for [a614768](https://github.com/googleinterns/av1-codec-comparison/commit/a614768fff380e51c98ce84511b406426f2a9209):
> 
> * I'm guessing "was not to decode" should be "was not able to decode"
> * Message should end with period.
> * SVT should be capitalized.

**Pull request merged**